src/paste.rs

```diff
@@ -181,9 +182,16 @@ impl Paste {
                     file_name = random_text;
                 }
             }
+            if let Some(no_ext) = random_url.no_extension {
+                if no_ext {
+                    no_extension = true;
+                }
+            }
```

Comment:
```suggestion
            no_extension = random_url.no_extension.unwrap_or(false);
```

Reply:
Nice. Yes, I missed that one. random_url is already unwrapped. 

In the past I had wrecked my brain how to do something like nested unwrap_or: `random_url.unwrap_or(what_the_heck_to_put_here).no_extension.unwrap_or(false);`